repository/src/main/java/org/apache/atlas/tasks/AtlasTaskService.java

```diff
@@ -27,21 +26,35 @@ public class AtlasTaskService implements TaskService {
 
     private final AtlasGraph graph;
 
+    private final List<String> retryAllowedStatuses;
+
+    static class SearchResult {
+        Long vertexTotals;
+        DirectIndexQueryResult directIndexQueryResult;
+
+        public SearchResult(Long vertexTotals, DirectIndexQueryResult directIndexQueryResult) {
+            this.vertexTotals = vertexTotals;
+            this.directIndexQueryResult = directIndexQueryResult;
+        }
+    }
+
```

Comment:
Let us not add this static class, this is anyway just helping to avoid repeating a single line which should not be a major issue if we revert

Reply:
Updated so that they both use methods which are overloaded. We do not need static SearchResult now